src/main/java/org/codeforamerica/shiba/output/caf/JobIncomeInformation.java

```diff
@@ -1,13 +1,22 @@
 package org.codeforamerica.shiba.output.caf;
 
+import org.codeforamerica.shiba.Money;
 import org.codeforamerica.shiba.pages.data.Iteration;
 
 public interface JobIncomeInformation {
     boolean isComplete();
 
-    Double grossMonthlyIncome();
+    Money grossMonthlyIncome();
 
     int getIndexInJobsSubworkflow();
 
     Iteration getIteration();
+
+    default Double parseWithCommasRemoved(String s) {
```

Comment:
Again I wonder if we should avoid using Double at all and stick to BigDecimal only

Reply:
+